### ISSUE TITLE
Ember Data model helper for integration tests

### ISSUE THREAD

**Issue by samselikoff** [samselikoff/ember-cli-mirage#1417]
![image](https://user-images.githubusercontent.com/2922250/48871220-c7177e00-edb1-11e8-96eb-aa4e93f4a906.png)

---

4 upvotes from Canny

**Comment by ro0gr** [comment]
just have seen it recently https://github.com/caseywatts/ember-proper-instantiation

**Comment by samselikoff** [comment]
@ro0gr yes exactly! Want to make something like this a first-class API in Mirage 👍

**Comment by flexyford** [comment]
Wanted to share a clickable link to @samselikoff gist: https://gist.github.com/samselikoff/b00b5a190321c8236ee0e0fab200bf65

We've extended the mirage `server` with `server.createRecord` and `server.createRecords` to return EmberData Models.  

I threw together a quick gist to show other what we've been working on. It's worked well for us so far. 
https://gist.github.com/flexyford/44ff73ec644ab9b80f1010ac4e30f674

Would love to bake this into mirage, let me know if these are steps in the right direction and if I can help in any way.

**Comment by samselikoff** [comment]
@flexyford awesome man 👌 , this is definitely one of the highest-priority items in Mirage. I'm focused on landing a 1.0 first (few more bugs but we're really close!)
